packages/office-ui-fabric-react/src/demo/pages/IconPage/examples/IconExample.scss

```diff
@@ -0,0 +1,56 @@
+.ms-IconExample {
+  font-size: 50px;
+  margin-left: 25px;
+  margin-right: 25px;
+}
+
+.ms-IconColorExample-deepSkyBlue {
+  color: deepskyblue;
+}
+
+.ms-IconColorExample-greenYellow {
+  color: #adff2f;
+}
+
+.ms-IconColorExample-salmon {
+  color: salmon;
+}
+
+.ms-IconImageSheetExample-one {
+  width: 48px;
+  height: 44px;
+  margin-left: 27px;
+}
+
+.ms-IconImageSheetExample-one-image {
+  display: inline-block;
+  position: relative;
+  left: -6px;
+  top: -4px;
+}
+
+.ms-IconImageSheetExample-check {
+  width: 35px;
```

Comment:
Can the width and height be specified in the image props, instead of added with this class?